packages/components/src/playground/button.tsx

```diff
@@ -12,6 +11,7 @@ export type ButtonProps = React.ButtonHTMLAttributes<HTMLButtonElement> &
 
 const ButtonComponent = styled.button<BaseButtonProps>(({ variant }) => [
   tw`py-2 px-4 rounded border-none`,
+  tw`font-arimo text-white font-bold text-sm leading-body`,
```

Comment:
Why are these in a separate item from `py-2 px-4 rounded border-none`? Are we grouping them logically?

Reply:
I was thinking first line is about shape/spacing and this line is about text. can leave comments

Reply:
Cool, that organization makes sense 👍